### ISSUE TITLE
sīquidem?

### ISSUE THREAD

**Issue by johnlawrenceaspden** [Alatius/latin-macronizer#14]
siquidem -> siquidem, which seems wrong or at least debatable?

L&S have: http://www.perseus.tufts.edu/hopper/morph?l=siquidem&la=la#lexicon

**Comment by Alatius** [comment]
Yeah, it's tricky in situations such as this to decide which form should be preferred. OLD describes the situation like this: "first syll. often short in comedy and regularly so in dactylic verse". I think I lean towards prefering the variant with short vowel for now, since that seems more probable to be correct in practice.
